kivy/core/window/__init__.py

```diff
@@ -933,6 +933,33 @@ def _set_top(self, value):
     the position set in :class:`~kivy.config.Config`.
     '''
 
+    def _get_opacity(self):
+        return self._get_window_opacity()
+
+    def _set_opacity(self, opacity):
+        return self._win.set_window_opacity(opacity)
```

Comment:
@pythonic64 and @mak8kammerer , here `return self._win.set_window_opacity(opacity)` should not be `self._set_window_opacity(opacity)` ?

Reply:
@misl6, look [this](https://github.com/kivy/kivy/pull/8296/files/0da1eabd3f922185c8a9c85eba697532062dd276#r1303461335)

Reply:
Seems that @pythonic64 was suggesting to change it to `return self._set_window_opacity(opacity)` (which is correct), and not `return self._win.set_window_opacity(opacity)` (at the link you provided)

Reply:
Sorry, my blame.

Reply:
@misl6, fixed! Thank You!

Reply:
@misl6 Correct, that line should be `return self._set_window_opacity(opacity)`